docker/example-healthcheck.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 if [ $CATENA_CONNECTION = "gRPC" ]; then
-    exec grpcurl -plaintext -proto st2138/service.proto -import-path st2138 localhost:$CATENA_PORT catena.CatenaService/GetPopulatedSlots >/dev/null 2>&1 || exit 1
+    exec curl -f -X POST --http2-prior-knowledge http://localhost:$CATENA_PORT || exit 1
 elif [ $CATENA_CONNECTION = "REST" ]; then
     exec wget -q --spider http://localhost:$CATENA_PORT/st2138-api/v1/health || exit 1
```

Comment:
```suggestion
    exec curl -f http://localhost:$CATENA_PORT/st2138-api/v1/health || exit 1
```
Forgot to update the REST healthcheck cause we took out wget, my bad